packages/inventory-compliance/src/ComplianceRemediationButton.js

```diff
@@ -57,15 +53,18 @@ class ComplianceRemediationButton extends React.Component {
         });
     }
 
+    noRemediationAvailable = () => {
+        const { allSystems } = this.props;
+        return !allSystems.some((system) => system.ruleObjectsFailed.some((rule) => rule.remediationAvailable));
```

Comment:
You need to take into account the second condition in the original conditional:

```
 selectedRules.length === 0
```

This happens on the system details page - in that page, `ruleObjectsFailed` is just `[]`, and we just select rules individually. Only `selectedRules` changes as one or more rules are selected, so right now since that's not taken into account, the button is always disabled on that page. It works great on all other pages though!